src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/list/AdminLocalAuthoritiesController.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.dts.fact.controllers.admin.list;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.hmcts.dts.fact.config.security.Role;
+import uk.gov.hmcts.dts.fact.exception.IllegalListItemException;
+import uk.gov.hmcts.dts.fact.model.admin.LocalAuthority;
+import uk.gov.hmcts.dts.fact.services.admin.list.AdminLocalAuthorityService;
+import uk.gov.hmcts.dts.fact.services.validation.ValidationService;
+
+import java.util.List;
+
+import static org.springframework.http.ResponseEntity.ok;
+import static uk.gov.hmcts.dts.fact.services.admin.AdminRole.FACT_ADMIN;
+import static uk.gov.hmcts.dts.fact.services.admin.AdminRole.FACT_SUPER_ADMIN;
+
+@RestController
+@RequestMapping(
+    path = "/admin/localauthorities",
+    produces = {MediaType.APPLICATION_JSON_VALUE}
+)
+public class AdminLocalAuthoritiesController {
+    private final AdminLocalAuthorityService adminLocalAuthorityService;
+    private final ValidationService validationService;
+
+    @Autowired
+    public AdminLocalAuthoritiesController(AdminLocalAuthorityService localAuthorityService, ValidationService validationService) {
+        this.adminLocalAuthorityService = localAuthorityService;
+        this.validationService = validationService;
+    }
+
+    @GetMapping(path = "/all")
+    @ApiOperation("Return all local authorities")
+    @Role({FACT_ADMIN, FACT_SUPER_ADMIN})
+    public ResponseEntity<List<LocalAuthority>> getAllLocalAuthorities() {
```

Comment:
This is the same as Majid's endpoint. Need to discuss how best to proceed with this. Ideally this endpoint is better to be in this class so we need to update the front end for Majid's stuff to point to this one, then remove Majid's endpoint and all associated tests

Reply:
Agreed. I will raise a new ticket in JIRA to cover moving the local authorities tab to use this new endpoint and remove the old one.